web/src/utils/state/atoms.ts

```diff
@@ -18,7 +18,7 @@ import {
 
 export const timeAverageAtom = atom<TimeAverages>(TimeAverages.HOURLY);
 
-const URL_TO_TIME_AVERAGE: Record<string, TimeAverages> = {
+export const URL_TO_TIME_AVERAGE: Record<string, TimeAverages> = {
```

Comment:
I'd like it to be a stepping stone to unifying `hourly` -> `24h` everywhere. Maybe I'm optimistic thinking having it here makes it more likely to be a short term thing but I think it would be nice in a different PR and do a full switch over for consistency.